jtune.py

```diff
@@ -1753,6 +1756,41 @@ def _at_exit(raw_gc_log=None, jmap_data=None, jstat_data=None,
                   optimized_for_ygcs_rate)
 
 
+def _generate_rotated_gc_log_name(gc_log_file, log_number):
+    return gc_log_file + "." + str(log_number)
+
+def _get_rotated_log_file(gc_log_file):
```

Comment:
There are conventions with preceding functions with underscores, but in jtune I try to have the convention that if it has side effects (where the function changes state of something else when the function is called), then I precede it with an underscore. If there are no side effects, it's better to not use the underscore, and make it more obvious to others that the function can be overridden.